contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -21,6 +21,12 @@
 logger = logging.getLogger(__name__)
 
 
+class ResponseType(object):
```

Comment:
This is confusing. Technically it is not a response type.

Reply:
Right it does seem confusing. I think it would be better to not refactor transport in this PR.